src/txmempool.h

```diff
@@ -672,10 +672,10 @@ class CTxMemPool
 class CCoinsViewMemPool : public CCoinsViewBacked
 {
 protected:
-    CTxMemPool &mempool;
+    const CTxMemPool &mempool;
 
 public:
-    CCoinsViewMemPool(CCoinsView *baseIn, CTxMemPool &mempoolIn);
+    CCoinsViewMemPool(CCoinsView *baseIn, const CTxMemPool &mempoolIn);
```

Comment:
I agree but was not aware that there is a general style preference of `uint256& hash` over `uint256 &hash`.  I personally like the former better, but did not want to reformat "someone else's" code.  Is the former explicitly preferred in Bitcoin sources?  (Which are not always consistent in style.)